ansible/roles/kubernetes/tasks/debian.yml

```diff
@@ -62,15 +62,7 @@
   retries: 5
   until: apt_install_helm is success
 
-# Adding required Kubernetes cgroups
```

Comment:
Is this no longer needed?

Reply:
The comment could have stayed but I think the task can be removed.